doc/guidelines/e2e_frontend_coding_guidelines.md

```diff
@@ -0,0 +1,132 @@
+# Frontend Coding Guidelines
+
+These guidelines will help you write frontend code that enables resilient and user-centric test execution.
+Currently these guidelines are strongly oriented according to Playwrights Best Practices.
```

Comment:
```suggestion
Currently these guidelines are strongly oriented towards Playwrights Best Practices.
```

I'm not entirely sure of this is the proper way of writing what you want to say. 
However, 'strongly oriented according to' sounds rather strange IMO.

Reply:
Probably oriented after is the phrase, shoudl rewrite that, thanks.